layouts/default.vue

```diff
@@ -15,12 +15,15 @@
 
     <!-- Footer-->
     <Footer :sections="footerSections"></Footer>
+
   </v-app>
 </template>
 
 <style lang="scss">
 .container {
   max-width: 1240px;
+  padding-left: 10px;
```

Comment:
am fine with this, but we might now want to subtract this off the manual padding we have on a bunch of our index.vue sections (which we should really factor out into a class)

Reply:
Yes, we need an app-wide spacing strategy and factor out any page-specific padding stuff going on right now.